examples/widgets/rangeWidget.js

```diff
@@ -0,0 +1,31 @@
+/* global itowns */
```

Comment:
no global variable and use import export, see comment for pick

rename file by RangeWidget.js